src/chainparams.cpp

```diff
@@ -235,8 +235,8 @@ class CMainParams : public CChainParams {
         strNetworkID = "main";
         //consensus.nSubsidyHalvingInterval = 210240; // Note: actual number of blocks per calendar year with DGW v3 is ~200700 (for example 449750 - 249050)
         consensus.nMasternodePaymentsStartBlock = 1; // not true, but it's ok as long as it's less then nMasternodePaymentsIncreaseBlock
-        consensus.nMasternodePaymentsIncreaseBlock = 2; // actual historical value
-        consensus.nMasternodePaymentsIncreasePeriod = 576*30; // 17280 - actual historical value
+        consensus.nMasternodePaymentsIncreaseBlock = 80000; // actual historical value
```

Comment:
"historical..." comment has been forgetten.

Reply:
Ok I will update that comment accordingly.